src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -665,7 +665,7 @@ static void fillQueryEntity(BSONArrayBuilder& ba, const EntityId* enP)
     ent.append(idString, enP->id);
   }
 
-  if (enP->type != "")
+  if ((enP->type != "") && (enP->type != "Thing"))  // FIXME PR: Is this OK (Thing)? (POST /v2/op/query needs it ...)
```

Comment:
I understand this approach makes "Thing" an alias of "any time". That is not the desired behaviour, i.e. a given user many query for entities which type is *exaclty* "Thing" and no other types.

Maybe defaulting should be applied by the upper layers (parsing, I guess) only in the case of "set" operations, i.e. create and update.